src/main/kotlin/no/nav/syfo/minesykmeldte/model/PreviewSykmeldt.kt

```diff
@@ -3,15 +3,15 @@ package no.nav.syfo.minesykmeldte.model
 import java.time.LocalDate
 
 // Når er man friskmeldt? 16 dager etter siste tom?
-data class Sykmeldt(
+data class PreviewSykmeldt(
     val narmestelederId: String,
     val orgnummer: String,
     val fnr: String,
     val navn: String,
     val startdatoSykefravaer: LocalDate,
     val friskmeldt: Boolean,
     val previewSykmeldinger: List<PreviewSykmelding>,
-    val soknader: List<Soknad>
+    val previewSoknader: List<PreviewSoknad>
 )
 
 // Type kan være "100%", "50%", "avventende", ++
```

Comment:
Kan sikkert fjerne de kommentarene mine, de var mest der for at vi skulle huske hva vi hadde snakket om i forhold til hva apiet skulle returnere :)